300-CNAB-security.md

```diff
@@ -0,0 +1,163 @@
+# Cloud Native Application Bundles Security (CNAB-Sec) 1.0 WD
+
+The CNAB Security specification augments the [CNAB Core specification](100-CNAB.md) by standardizing on security mechanisms for signing, verifying, and attesting CNAB packages. It describes both a client/registry security model and a verification chain (provenance) model. By design, this provides security mechanisms that work in air-gapped networks.
+
+This specification is distinct from the CNAB Core specification. An implementation may comply with the CNAB Core specification, and yet not comply with this specification. The use of terms such as MUST and SHOULD in this document are statements about how a CNAB implementation may fulfill the CNAB Security specification _only_.
```

Comment:
nit: consistent casing for verbs - here, _An implementation [MAY?] comply_.

Reply:
There is a difference between `may` and `MAY`, since MAY means "as a matter of conforming to this specification, something is allowed to..." while "may" here is non-normative -- as in "it is possible that something could comply to another specification...".

Reply:
That was like saying, "I see your nit, and raise you a second nit!"